k8s-resources.tf

```diff
@@ -42,6 +42,12 @@ resource "kubernetes_cluster_role" "alb_ingress_controller" {
   }
 }
 
+resource "kubernetes_secret" "alb_ingress_controller" {
+  metadata {
+    name = "alb-ingress-controller"
```

Comment:
I'm surprised you didn't have to set the type of this secret to `kubernetes.io/service-account-token` and the `kubernetes.io/service-account.name` annotation to the service account name